src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private Color [] color = Color.values();
+    private Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        return color[random.nextInt(color.length)].toString();
```

Comment:
```suggestion
        return color[random.nextInt(color.length)].name();
```
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/9db38520-679b-4155-b64a-319c1e123e97)

Reply:
Теперь я попробовал и понятно, а то я думал вместо слова name нужно что-то подставить. В видео об этом не было вроде ни слова.
 private Color [] color = Color.values(); цю змінну можна було не робити вона зайва?